newIDE/app/src/MainFrame/EditorContainers/HomePage/index.js

```diff
@@ -41,6 +41,7 @@ import PreferencesContext from '../../Preferences/PreferencesContext';
 import { type FileMetadataAndStorageProviderName } from '../../../ProjectsStorage';
 import generateName from '../../../Utils/ProjectNameGenerator';
 import { sendTutorialOpened } from '../../../Utils/Analytics/EventSender';
+import { arePendingNotifications } from '../../../Utils/Notification';
```

Comment:
nit: `hasPendingNotifications` or `areNotificationsPending`?

Reply:
Yeah, better

Reply:
(The first 😅 )